Source/WebCore/html/HTMLTableCellElement.cpp

```diff
@@ -59,8 +59,11 @@ unsigned HTMLTableCellElement::colSpan() const
 
 unsigned HTMLTableCellElement::rowSpan() const
 {
-    // FIXME: a rowSpan equal to 0 should be allowed, and mean that the cell is to span all the remaining rows in the row group.
-    return std::max(1u, rowSpanForBindings());
+    unsigned rowSpanValue = rowSpanForBindings();
+    // when rowspan=0, the specs says it should apply to the full rows.
+    if (!rowSpanValue)
+        return maxRowspan;
```

Comment:
Chrome returns 1 in this case: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/html/html_table_cell_element.cc;l=55?q=HTMLTableCellElement::rowSpan&ss=chromium

Reply:
I'm not sure what Chrome is doing after with this one. because `rowspan="0"` is **not** the equivalent of `rowspan="1"`

![Screenshot 2025-01-07 at 13 31 52](https://github.com/user-attachments/assets/118d4d20-bee5-4a48-a642-f4ea8a2731d7)

Reply:
@nt1m they do something similar in fact. 
Just found.
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/layout/table/layout_table_cell.h;l=29-32;drc=f39c57f31413abcb41d3068cfb2c7a1718003cc5;bpv=1;bpt=1